server.js

```diff
@@ -1,8 +1,19 @@
+require("dotenv").config();
 const express = require("express");
 const app = express();
+const mongoose = require("mongoose");
 
 app.get("/", (req, res) => res.send("Hello World!"));
 
+mongoose.connect(process.env.DATABASE_URL, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+
+const db = mongoose.connection;
+db.on("error", (error) => console.error(error));
+db.once("open", () => console.log("connected to Database"));
```

Comment:
```suggestion
db.once("open", () => console.log("Connected to database"));
```
Minor: Let's try and write in [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)

Reply:
Done! :)

Reply:
Done! :)